pool/app/session/session.mli

```diff
@@ -236,6 +255,12 @@ val has_bookable_spots_for_experiments
   -> Experiment.Id.t
   -> (bool, Pool_common.Message.error) result Lwt.t
 
+module Repo : sig
```

Comment:
I didn't find the usage outside of the module, where this is used.
Can this be removed?